0002/README.md

```diff
@@ -0,0 +1,197 @@
+subprojects will have an intrinsic release target if they want to get their new
+code included in the Tinkerbell release.
+
+`PATCH` release for Tinkerbell can be done as required by the subprojects
+bug fixes.
+
+`MAJOR` release right now are not even evaluated because the project is too
+young. By definition breaking changes require a new `MAJOR` release, right now
+any of our projects have a release yet, we are working hard to reach `v1.0.0`,
+until there bc breaks are allowed but they will be notified with a proper label
+`bc-break` attached to the pull request.
+
+### GitHub release and release website
+
+Git supports tag and GitHub has the concept of a release based on a particular
+tag.
+
+The artifact generated as part of the release will be published as part of the
+release website, using as a guidance how [HashiCorp
+Release](https://releases.hashicorp.com/) works.
+
+I like the idea to use a release website vs having to push them as manifest in
+GitHub release because it gives a single place where the community can
+programmatically lookup what they are looking for, without having to look for
+repositories, versions and so on.
+
+As part of the Release page on GitHub we will publish a generated changelog.
+
+### Distribution
+
+Docker containers, binaries, packages such as `deb`, `rpm` requires
+infrastructure such as:
+
+1. A place where the binaries should be archived and easy do be downloaded from
+   (bintray, s3)
+2. Docker containers requires a registry
+
+In order to gain consistency across projects and to build familiarity for the
+community we will start as follow:
+
+1. Docker containers will be pushed to `quay.io` under the `tinkerbell`
+   organisation (this is already done)
+2. Binaries and possibly `rpm`, `deb` will be pushed to
+   [Bintray](http://bintray.com/) because it has a friendly open source plan.
+
+Right now there is not a formal group or structure that manages the Tinkerbell
+community needs in terms of infrastructure, servers and SaaS. We are building
+this group but nothing is formalized yet. As soon as the group will be assembled
+they will be the right target for questions, and requests about bintray, s3,
+Packet devices. In the meantime you can ask on
+[Slack](https://tinkerbell.org/community/slack/) in the #tinkerbell channel.
+
+### Architectures and operating systems
+
+The majority of the projects in the tinkerbell organisation right now are in Go,
+it means that it is easy to compile targeting different architectures and
+operating system. We also have an application in JavaScript (portal) and an
+operating system distribution (osie). It is not part of the this proposal to
+decide what Tinkerbell should support or not. But as a direction all the project
+should setup their release workflow in a way that will support:
+
+1. Multiple operating systems and architectures
+2. Static binaries and artifacts
+3. Docker containers
+4. APT, DEB, RPM package manager.
```

Comment:
apt and deb are the same, like dnf to rpm.

Perhaps we should remove all deb/rpm requirements for now.

We are not prepared to build, host (could be on launchpad, koji, etc), or maintain (install/uninstall scripts, dependency requirements) these artifacts.